setup.py

```diff
@@ -53,14 +53,25 @@ def _get_version_from_file(filename, start_of_line, split_marker):
     long_description=open('README.rst').read(),
     classifiers=[
         'Intended Audience :: Developers',
+        'Operating System :: OS Independent',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
```

Comment:
This seems duplicate (see line 57 above), and maybe (just for ordering reasons) the 2.7 classifier should be moved before the :: 3 classifier.

Reply:
Ah, good call. I'll fix this one sec